src/common/mClockCommon.h

```diff
@@ -0,0 +1,29 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2017 SK Telecom
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#pragma once
+
+#include "dmclock/src/dmclock_recs.h"
+
+// the following is done to unclobber _ASSERT_H so it returns to the
+// way ceph likes it
+#include "include/assert.h"
+
+
+namespace ceph {
+  namespace dmc = crimson::dmclock;
+}
+
+WRITE_RAW_ENCODER(dmc::ReqParams)
+WRITE_RAW_ENCODER(dmc::PhaseType)
```

Comment:
as i put in last comment, this line is not necessary.

Reply:
I rolled back it because eliminating the line cause compile errors. To my understanding, denc for enum only supports uint8_t, int8_t and several underlying types.(BTW, is it intended?) But PhaseType in current dmclock code uses the default underlying type(unsigned int).
I requested to change the underlying type for PhaseType with new PR on dmclock github project.(https://github.com/ceph/dmclock/pull/45)
For now, keeping current code would be good for this PR.
I will eliminate that line next time after the dmclock PR is merged into ceph.

Reply:
oh, my bad. i didn't realize that `PhaseType` is an `unsigned`.